components/intake-form/WorkExperience.tsx

```diff
@@ -18,6 +18,12 @@ import Selectable, {
   SelectableVariant,
 } from "../form/Selectable";
 
+export const labelNote =
```

Comment:
I usually would use this sort of pattern if the string is replaced elsewhere. Any reason we donʻt just refer to this string below? Iʻm also wondering if that makes these backslashes necessary.

Reply:
Answering this question, while also answering your other question "_any reason we donʻt do the same for industries?_"

I do use this `labelNote` in 03-company.tsx where the same note is used for industry suggestions because we're not actually using the [CompanyIndustry](https://github.com/hawaiians/hawaiiansintech/blob/76a062372ab3523cacf859c091c54db5956e9602/components/intake-form/CompanyIndustry.tsx#L35) function in [03-company.tsx](https://github.com/hawaiians/hawaiiansintech/blob/main/pages/join/03-company.tsx). But let me know if we should be.

I added backslashes to keep the string from being too long. For some reason the linter wasn't formatting it for the recommended 80 char limit so I did it manually. Let me know if there's a better way.

Reply:
Got it. Makes sense. I could see us creating a new component `WorkExperienceWarning.tsx` or something to import around. But where, geez idk. Another route is appending it to `WorkExperience.tsx`, I suppose.

On the string thing, up to you. You might(?) be able to use template literal backticks "`" around the string... and it should let you be more liberal with line breaks and stuff? Anyway, if that doesn't work or if you'd rather ignore, no problem mon.

Reply:
Done with the latest commit